functions/http/index.js

```diff
@@ -148,20 +148,31 @@ exports.uploadFile = (req, res) => {
   if (req.method === 'POST') {
     const busboy = new Busboy({ headers: req.headers });
 
+    // This object will accumulate all the fields, keyed by their name
+    const fields = {};
+
     // This object will accumulate all the uploaded files, keyed by their name.
     const uploads = {};
     const tmpdir = os.tmpdir();
 
-    // This callback will be invoked for each file uploaded.
+    // This event will be triggered for each non-file field in the form.
```

Comment:
I'm a bit afraid that users may confuse events described above, with event that trigger a function. For instance, users may think that there are many function executions here.

I think the previous form ("This callback will be invoked") was a bit better

Also, busboy docs say that events are emitted, not triggered.

How about something like this:

- "This code is executed on..."
- "This code handles busboy events emitted on..."
- "This busboy events are emitted on..."